ssz_serialization/digest.nim

```diff
@@ -47,11 +47,18 @@ when PREFER_HASHTREE_SHA256 and (defined(arm64) or defined(amd64)) and (
   (defined(macosx) and defined(clang) and defined(arm64))
 ):
   {.hint: "Hashtree SHA256 backend enabled".}
-  const USE_HASHTREE_SHA256 = true
 
-  import ../vendor/hashtree/hashtree_abi
-else:
-  const USE_HASHTREE_SHA256 = false
+  when tryImport ../vendor/hashtree/hashtree_abi:
+    {.hint: "hashtree_abi found in vendor(nimbus build system)".}
```

Comment:
`vendor` isn't intrinsically linked to `nimbus-build-system`, it's just that `nimble` happens not to understand (intentionally or not, I don't know) git submodules. But this phrasing assumes something potentially untrue, I've personally routinely tested `nim-ssz-serialization` changes in isolation, by ensuring I'd gotten submodules.